jobs/pull-kubernetes-node-e2e-cri.sh

```diff
@@ -1,46 +0,0 @@
-#!/bin/bash
-# Copyright 2016 The Kubernetes Authors.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-set -o errexit
-set -o nounset
-set -o pipefail
-set -o xtrace
-
-case "${PULL_BASE_REF:-}" in
-release-1.0|release-1.1|release-1.2|release-1.3|release-1.4)
```

Comment:
FYI this is important logic that was lost. https://github.com/kubernetes/kubernetes/pull/40552